include/envoy/router/router.h

```diff
@@ -683,6 +716,13 @@ class RouteEntry : public ResponseEntry {
    */
   virtual const RetryPolicy& retryPolicy() const PURE;
 
+  /**
+   * @return const InternalRedirectPolicy& the internal redirect policy for the route. All routes
+   *         have a internal redirect policy even if it is not enabled, which means redirects from
+   *         the upstream are not followed.
+   */
```

Comment:
redirects from the upstream are not followed.
->
redirects are simply proxied as normal responses
?

Reply:
Done. Somehow missed this one.